lblod/app-lokaal-mandatenbeheer
# Refactor mandataris forms by using form extensions

## Description

Reused the common logic in mandataris forms and made them extend each other. Mandataris-new now extends mandataris-edit and mandataris-ext extends mandataris-new.
All mandataris forms now have a link-to-besluit form. We probably don't want this, but we need to see how we want to use the link-to-besluit.
Also removed the bestuursorgaan-ext forms, since they where a proof of concept and we now actually extend forms in a useful way.
 
## How to test

Open a mandataris-new, mandataris-edit and mandataris-ext form and see if they still work.
Be aware, you do need the form-helpers fix for this!
